src/infrastructure/TransactionService.js

```diff
@@ -337,6 +324,15 @@ class TransactionService {
   			action: Constants.MosaicSupplyChangeAction[transactionBody.action],
   			delta: transactionBody.delta.compact()
   		};
+  	case TransactionType.MOSAIC_SUPPLY_REVOCATION:
+  		return {
+  			transactionType: transactionBody.type,
+  			address: transactionBody.sourceAddress.address,
```

Comment:
Maybe we should add more details about the address. We can leave it as `sourceAddress`

Reply:
yes, they prefer to display `address` rather than source address. check out the discord chat record I shared on top.